frontend/src/utils/TaskUtils.ts

```diff
@@ -245,6 +245,8 @@ export const calcTaskWeightedValueSum = (
   allCustomers: Customer[],
   roadmap: Roadmap,
 ) => {
+  if (!allCustomers) return;
```

Comment:
The type is `Customer[]`, which will never be falsy, so the correct fix would either be:
- also change the parameter type
- check the parameter at call site

This would also change the return type from `number` to `number | undefined`, which would require additional handling at the call site.

I would suggest that the "correct" fix would be checking the parameters in the caller (e.g. `customers ?? []`, if the caller doesn't care about undefined customers)